maestro-cli/src/main/java/maestro/cli/view/TestSuiteStatusView.kt

```diff
@@ -63,11 +69,11 @@ object TestSuiteStatusView {
                 )
 
                 if (canceledFlows.isNotEmpty()) {
-                    println("(${canceledFlows.size} ${flowWord(canceledFlows.size)} Canceled)")
+                    PrintUtils.warn("${canceledFlows.size} ${flowWord(canceledFlows.size)} Cancelled (Internal Maestro Cloud Error)")
```

Comment:
What happens in the event of timeouts or skipped Flows due to dependency Flows fail? Will it say `Internal Maestro Cloud Error` then even though this is not the case?

Reply:
> What happens in the event of timeouts or skipped Flows due to dependency Flows fail

Yes, there's no way to know the difference client side atm

Reply:
@axelniklasson Removed the Internal Error message as it might cause more confusion.